hbase-server/src/main/java/org/apache/hadoop/hbase/util/RegionMover.java

```diff
@@ -181,6 +184,19 @@ public RegionMoverBuilder(String hostname, Configuration conf) {
       } else {
         this.port = conf.getInt(HConstants.REGIONSERVER_PORT, HConstants.DEFAULT_REGIONSERVER_PORT);
       }
+
+      // if use-ip is enabled, we will resolve the hostname to ip to ensure that the RegionMover
+      // logic can be executed normally, see HBASE-27304 for details.
+      boolean useIp = conf.getBoolean(HConstants.HBASE_SERVER_USEIP_ENABLED_KEY,
```

Comment:
This is a tool? Can we get the hostname/ip by accessing the hbase cluster, instead of relying on the configuration? What if the configuration mismatches?

Reply:
@Apache9  sir.
Thanks for your review. 
There may be configuration mismatch. I have modified and submitted the code.
Could you take a look? Thanks.